### DISCUSSION TITLE
Add tag "hacktoberfest" to repository for contributions

### DISCUSSION THREAD

**Discussion by baricua** [iusehooks/usetheform#46] [General]
I like the project and found some points of improvement for the documentation. I would like to contribute to it in view of the currently ongoing Hacktoberfest 2023, but the repositories need to be tagged with the "hacktoberfest"  topic in order for the contribution to be accepted by the fest. You can read more about the requirements [here](https://hacktoberfest.com/participation/). I hope you'll take it into consideration, I would be happy to start working on it!

**Comment by iusehooks** [comment]
Hi @Victoria-Bruno , topic hacktoberfest and hacktoberfest2023 have been added . Thanks
